### ISSUE TITLE
[xml import] find a way to match XMLs even if the filenames changed

### ISSUE THREAD

**Issue by sennierer** [acdh-oeaw/apis-instance-oebl-pnp#247]
eg in [this version history](https://oebl-pfp.acdh-ch-dev.oeaw.ac.at/apis/history/change_history/apis_ontology.person/89241/) the print edition version is missing because the filename changed from `Barsescu_Agathe_1861_.xml` (print edition) to `Barsescu_Agathe_1859_1939.xml` for the online edition. Given that IDs are unfortuntely missing in the GIDEON exports we need to match (parts of) the publication string, the name and the birth date.

**Comment by b1rger** [comment]
Is this something we can do *after* the import?

**Comment by sennierer** [comment]
no, actually I am working on doing that within the XML harmonization script
